src/components/Card/Card.tsx

```diff
@@ -11,18 +12,41 @@ function Card({
   flip: boolean;
   onClick: () => void;
 }) {
+  const ref = React.useRef<HTMLDivElement>(document.createElement("div"));
+
+  React.useEffect(() => {
+    let entryPoint = [0, 0];
+    const track = (moveEvent: MouseEvent) => {
+      moveEvent.stopPropagation();
+      ref.current.style.transform = `translate(${
```

Comment:
you might want to use `const el = ref.current` to avoid repetition. el/div - naming up to you. But the name should give a better idea about what we're working with